enterprise/server/backends/userdb/userdb.go

```diff
@@ -513,8 +513,12 @@ func (d *UserDB) GetGroupUsers(ctx context.Context, groupID string, statuses []g
 			FROM "Users" AS u JOIN "UserGroups" AS ug ON u.user_id = ug.user_user_id`)
 	q = q.AddWhereClause(`ug.group_group_id = ?`, groupID)
 
+	if opts.SubIDPrefix != "" {
+		q = q.AddWhereClause("u.sub_id LIKE ?", opts.SubIDPrefix+"%")
```

Comment:
is performance of this a concern?

Reply:
I don't think so since this filter will happen on top of the existing group ID filter.